### PR TITLE
implemented second section of landing page

### PR DESCRIPTION
**Pull request by mevlutcantuna**
This changes for #62 Implement new landing page second section according to new ui design.

The background color will be changed when the first section is changed.

### CODE REVIEW

File: gurubu-client/src/app/components/page/testimonial.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
+
+const Testimonial = () => {
+  return (
+    <section className="testimonial-section">
+      <div className="testimonial">
+        <Image
+          width={123}
+          height={27}
```
Comment (mertcanaltin) [inline gurubu-client/src/app/components/page/testimonial.tsx:9]: we can enlarge our logo like in figma
```suggestion
          width={140}
          height={40}
```

File: gurubu-client/src/app/styles/page/testimonial.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            flex-direction: column;
+
+            &__name{
+                margin: $space-large 0 $space-xsmall 0;
+                font-size: $font-size-paragraph-2;
+                font-weight: $semibold;
+            }
```
Comment (mertcanaltin) [inline gurubu-client/src/app/styles/page/testimonial.scss:37]: here we can make the color the same as figma
```suggestion
            }
            &__title{
                color: #475467;
            }
```

**Comment by mertcanaltin** [comment]
LGTM! It just needs a few minor edits, thank you so much for taking care of it 🚀

**Review by mertcanaltin — DISMISSED** [review]

**Comment by armagandalkiran** [comment]
Thank you for your contribution! LGTM too. 3 more section to go for this epic, after that we can merge this pull request!

File: gurubu-client/src/app/components/page/testimonial.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+          width={140}
+          height={40}
+          alt="grubu-logo"
+          src="/gurubu-logo.svg"
+        />
+        <p className="testimonial__text">
+          I was looking for an application that could quickly determine story
+          points without registration, and I finally found it.
+        </p>
+
```
Comment (armagandalkiran) [inline gurubu-client/src/app/components/page/testimonial.tsx:17]:
```suggestion
```

File: gurubu-client/src/app/page.tsx [hunk 4]
```diff
@@ -10,6 +11,7 @@ export default function Home() {
       <Navbar />
       <Greeting />
       <Howto />
+      <Testimonial />
```
Comment (mertcanaltin) [inline gurubu-client/src/app/page.tsx:14]: @armagandalkiran Would it be healthy to temporarily merge this into the comment line?
```suggestion
      //<Testimonial />
```

**Review by mertcanaltin — DISMISSED** [review]
looks good 🚀

**Review by aahmetcakir — APPROVED** [review]

**Review by armagandalkiran — APPROVED** [review]

**Comment by mertcanaltin** [comment]
congratulations and we thank you very much for taking care of this issue 🎉

**Comment by mevlutcantuna** [comment]
Thank you so much, if I have more time, I would like to get more issues 😃

**Pull request merged**